packages/at_secondary_server/lib/src/verb/handler/sync_progressive_verb_handler.dart

```diff
@@ -9,10 +9,14 @@ import 'package:at_secondary/src/verb/handler/abstract_verb_handler.dart';
 import 'package:at_secondary/src/verb/verb_enum.dart';
 import 'package:at_server_spec/at_server_spec.dart';
 import 'package:at_server_spec/at_verb_spec.dart';
+import 'package:meta/meta.dart';
 
 class SyncProgressiveVerbHandler extends AbstractVerbHandler {
   static SyncFrom syncFrom = SyncFrom();
 
+  @visibleForTesting
+  ByteBuffer syncBuffer = ByteBuffer(capacity: AtSecondaryConfig.syncBufferSize);
```

Comment:
Can this be an issue when dealing with async and await behavior?

Reply:
Resolved this comment. Moved the sync buffer into the processVerb method.